rest_framework-stubs/serializers.pyi

```diff
@@ -225,7 +225,7 @@ class ModelSerializer(Serializer, BaseSerializer[_MT]):
     def update(self, instance: _MT, validated_data: Any) -> _MT: ...
     def create(self, validated_data: Any) -> _MT: ...
     def save(self, **kwargs: Any) -> _MT: ...
-    def to_representation(self, instance: _MT) -> Any: ...
+    def to_representation(self, instance: _MT) -> dict[str, Any]: ...
```

Comment:
Looks like `ModelSerializer` doesn't override this method, we can simply remove and it will be inherited.
```suggestion
```